braindecode/augmentation/transforms/masking_along_axis.py

```diff
@@ -0,0 +1,138 @@
+import torch
```

Comment:
I would avoid a new nested directory here and put files in the augmentation module.
Doing

from braindecode.augmentation.transforms import ...

seems too much. 

also maybe braindecode.augment could be enough